src/tests/unit/tests/DetailsView/components/left-nav/left-nav-link-builder.test.tsx

```diff
@@ -327,14 +327,14 @@ describe('LeftNavBuilder', () => {
     ): TestRequirementLeftNavLink {
         return {
             name: requirement.name,
-            key: requirement.key,
```

Comment:
Shouldn't we still validate the key here?

Reply:
Wanted to remove it for whenever we use what you end up working on ... but I guess I'll re-add it until then.